pkg/stis/calstis/cs2/cs2.c

```diff
@@ -23,7 +23,8 @@ int main (int argc, char **argv) {
 
 	c_irafinit (argc, argv);
 
-	if ((input = calloc (strlen(argv[1]) + 1, sizeof(char))) == NULL) {
+	/* ~1Kb padding per path, to allow for string modifications later */
+	if ((input = calloc(argc * (STIS_FNAME * 2), sizeof(char))) == NULL) {
```

Comment:
I agree, Phil, this calloc should just allocate the length of the input string and nothing more. Jamie -- yeah exactly. Sorry for the confusion 😬. 

The massive calloc was an attempt to brute-force "success" by allocating far more memory than required (in a silly way no less). Of course, this lead to some interesting discoveries within HSTIO's error handling routines, but that's about it.

I've been digging through the POSIX docs as well as various filesystem comparison charts all night trying to figure out a good way to handle path/file length limits. I think it may be possible to consolidate some of SZ_* definitions into a common place (either a header or dynamically generate them at run-time) and not adversely effect anything.